MoneySplitter.Services/Defines.cs

```diff
@@ -2,6 +2,10 @@
 {
     public static class Defines
     {
+        public static class Executor
+        {
+            public const int COUNT_TRY = 3;
```

Comment:
rename to somehting like "EXECUTION_ATTEMPTS_COUNT"

Reply:
rename